packages/mobile/README.md

```diff
@@ -176,6 +176,12 @@ Where `YOUR_BUILD_VARIANT` can be any of the app's build variants, such as debug
 We're using [GraphQL Code Generator][graphql code generator] to properly type
 GraphQL queries. If you make a change to a query, run `yarn build:gen-graphql-types` to update the typings in the `typings` directory.
 
+## Running Wallet app in ZeroSync mode
+
+By default, the mobile wallet app runs geth in ultralight sync mode where all the epoch headers are fetched. The default sync mode is defined in [packages/mobile/.env](https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/.env#L4) file.
```

Comment:
Nit, we've placed most links at the bottom of the readme

Reply:
I see in-line links everywhere in this file?